CHANGELOG.md

```diff
@@ -1 +1,2 @@
 - Improves login flow when using the `--reauth` flag.
+- Increases minimum Node.js version to v8.6.0, accurately reflecting underlying requirements.
```

Comment:
Maybe say what requires 8.6 - I would hate for people to think other things (CF3) had this change.

```suggestion
- Increases minimum Node.js version to v8.6.0, accurately reflecting underlying requirements of the CLI tool.
```